carma-messenger-ui/website/widgets/emergencyResponse/widget.js

```diff
@@ -4,6 +4,22 @@
 CarmaJS.registerNamespace("CarmaJS.WidgetFramework.emergencyResponse");
 var listenerAlert;
 var listenerBSM;
+var map = null;
+const ERV_ROUTE_SOURCE = "erv-route-trace";
+var data = {
```

Comment:
Can we add comments to describe `data` and `destination_pin`?

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/177/commits/89771b758699a93896e5555ceb112d4a9e320309